### PR TITLE
Prebid 10: normalize EID and schain in FPD; move legacy schain configuration logic to schain module

### PR DESCRIPTION
**Pull request by dgirardi**
## Type of change

- [x] Feature

## Description of change

 - Normalize EIDs and schain so that bid adapters always find them in user.ext.eids and source.ext.schain respectively
 - EIDs are concatenated and deduped. Schain is picked preferentially from source.schain, with a warning if it conflicts with source.ext.schain
 - The legacy schain configuration is now supported by the schain module (and not core)

## Other information

Related: https://github.com/prebid/Prebid.js/pull/13264/

### CODE REVIEW

File: test/spec/fpd/normalize_spec.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          eids: [{source: 'id'}],
+          ext: {eids: [{source: 'id'}]}
+        }
+      }
+      expect(normalizeEIDs(fpd).user.ext.eids).to.eql([
+        {source: 'id'}
+      ])
+      sinon.assert.called(utils.logWarn);
+    });
+    it('should NOT remove duplicates if they come from the same place', () => {
```
Comment (patmmccann) [inline test/spec/fpd/normalize_spec.js:41]: interesting edge case :)
